.setup/data/courses/tutorial.yml

```diff
@@ -17,7 +17,8 @@ gradeables:
     g_grade_start_date: +7 days
     g_grade_due_date: +9 days
     g_grade_released_date: +9 days
-    eg_regrade_request_date: +16 days
+    eg_regrade_request_start_date: +16 days
```

Comment:
generally the start date for grade inquiries should be the day that the grades are released.  
Sometimes the instructor will want a slight pause between release of grades and start of grade inquiries, but this will probably be 1 or 2 days only.

For the sample assignments set the difference to be 1 day (that is, set the new start date for grade inquiries to be the grades released deadline + 1 day), and leave the deadline for submitting grade inquiries as it was.